functions/__sf_section_julia.fish

```diff
@@ -0,0 +1,37 @@
+#
```

Comment:
It looks like there might be inconsistent indentation on this page. 🤔

Reply:
It appears to me that most lines before `__sf_lib_section` use spaces rather than tabs.

Reply:
😯